src/ProjectTemplates/Web.ProjectTemplates/content/Components-CSharp/Pages/FetchData.razor

```diff
@@ -0,0 +1,56 @@
+﻿@page "/fetchdata"
+
+<PageTitle>Weather forecast</PageTitle>
+
+<h1>Weather forecast</h1>
+
+<p>This component demonstrates fetching data from the server.</p>
+
+@if (forecasts == null)
+{
+    <p><em>Loading...</em></p>
+}
```

Comment:
This page never goes async so this will never actually do anything right?

Reply:
Yeah, we can simplify this part. Alternatively, we could do a task.yield on the service to generate asynchrony and showcase streaming rendering.

Reply:
That's an interesting idea. To really see the effect I think a `Task.Delay()` is needed, so that the "loading" message is visible in the browser before being updated with the data.

Reply:
I'm in favor of enabling streaming rendering on the weather forecast page and including a brief delay to showcase it. But then I also want an empty option on this template 🙂

Reply:
Ugh, no. I'd rather just do the empty template if that's the case.

Reply:
OK, how about we enable streaming rendering, but don't add the delay? The developer can then add the delay or a real API call to see the effect.

Reply:
I don't think there's value in doing that, since the component is going to render synchronously anyway. There won't be any difference.

Reply:
The weather forecast page exists to demonstrate a page that gets some data from somewhere and displays it. Typically retrieving that data will involve an async operation and streaming rendering will be relevant. Assuming that the data will be retrieved synchronously seems like a special case optimization for the template, when we really should be showing the more common pattern.

Reply:
Then, I think we should have a `Task.Delay` to demonstrate it. Otherwise, it's just confusing. Will that work for you?

Reply:
Yup!